src/components/topbar/TopbarTimer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { 
+  selectPlaylistPlaying,
+  PlayPausePlaylistState
+} from '../../state/slices/playPausePlaylistSlice'
```

Comment:
we could set the `baseUrl = "src"` in `tsconfig` to avoid having to do any `../../` action 😉

Reply:
does this need any vite configuration..? i think with webpack it might have been needed (?)